classes/options-general.php

```diff
@@ -624,14 +624,14 @@ class='howto'><?php _e( 'Select "No Limit" to display all results within the sea
 																$safe   = str_replace( '-', '_', $taxonomy );
 																$label  = isset( $tax_info['description'] ) ? $tax_info['description'] : str_replace( 'sm-', '', $taxonomy );
 																$active = ! empty( $options['taxonomies'][ $taxonomy ] );
-																echo '<label for="taxonomies_' . $safe . '"><input type="checkbox" name="taxonomies[' . $taxonomy . '][active]" id="taxonomies_' . $safe . '" ' . checked( $active, true, false ) . ' /> ' . __( 'Enable ' . $label . ' taxonomies?', 'simplemap' ) . '</label>';
+																echo '<label for="taxonomies_' . $safe . '"><input type="checkbox" name="taxonomies[' . $taxonomy . '][active]" id="taxonomies_' . $safe . '" ' . checked( $active, true, false ) . ' /> ' . __( 'Enable ' . $label . ' taxonomies?', 'SimpleMap' ) . '</label>';
 																echo '<br />';
 																if ( $active && ! isset( $standard_taxonomies[ $taxonomy ] ) ) {
 																	echo '<div style="margin: 5px 0 5px 15px; padding: 5px; border: 1px solid #ccc;">';
-																	echo '<label for="taxonomies_' . $safe . '_singular">' . __( 'Singular Form', 'simplemap' );
+																	echo '<label for="taxonomies_' . $safe . '_singular">' . __( 'Singular Form', 'SimpleMap' );
 																	echo ': <input type="text" name="taxonomies[' . $taxonomy . '][singular]" id="taxonomies_' . $safe . '_singular" value="' . esc_attr( $tax_info['singular'] ) . '" /></label>';
 																	echo '<br />';
-																	echo '<label for="taxonomies_' . $safe . '_plural">' . __( 'Plural Form', 'simplemap' );
+																	echo '<label for="taxonomies_' . $safe . '_plural">' . __( 'Plural Form', 'SimpleMap' );
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 156 characters

<!-- Comment by [SideCI](https://sideci.com) -->